services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/cache/ArionWingCache.java

```diff
@@ -33,7 +33,10 @@ free of charge, to any person obtaining a copy of this software and associated d
 @ComponentScan(value="com.futurewei.alcor.common.db")
 public class ArionWingCache {
 
+    // arionWingCache store Arion wing meta data. key is Arion Wing hash code and value is Arion wing meta data.
     private ICache<String, ArionWing> arionWingCache;
+
+    // arionWingGroupCache store Arion group meta data. key is Arion wing group name, value is Arion wing meta data.
```

Comment:
Is the value here Arion group info? What is the difference from ArionWing which is the wing meta data.

Reply:
There might be several Arion Wing belong to one group